typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -84,7 +76,7 @@ export class EvmERC20WarpModule extends HyperlaneModule<
 
     return new EvmERC20WarpModule(multiProvider, {
       addresses: {
-        deployedTokenRoute: deployedContracts[config.type].address,
+        [config.type]: deployedContracts[config.type].address,
```

Comment:
does this make type-ing harder to reason about?

Reply:
reverted